pkg/consts/breakline/breakline.go

```diff
@@ -1,9 +1,12 @@
-// Package contains all breakline strategies.
+// Package breakline contains all break line strategies.
 package breakline
 
+// Strategy represents a break line strategy.
 type Strategy string
 
 const (
+	// EmptyLineStrategy is a break line strategy that uses empty lines.
```

Comment:
My error here. Actually this should be EmptySpaceStrategy since it uses the spaces between words... Could you rename it?

Reply:
// EmptySpaceStrategy is a break line strategy that counts the length of words to create a new line. This strategy only works in languages that use spaces to divide words.